backend/routes/auth.js

```
@@ -0,0 +1,154 @@
+},
+    async function (accessToken, refreshToken, profile, done) {
+        const newProvider = {
+            name: 'google',
+            providerId: profile.id
+        };
+        const newUser = {
+            id: uuidv4(), // Generate a unique id
+            displayName: profile.displayName,
+            firstName: profile.name.givenName,
+            lastName: profile.name.familyName,
+            email: profile.emails && profile.emails[0] ? profile.emails[0].value : '',
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)